api/app.js

```diff
@@ -18,23 +19,21 @@ app.use(logger('dev'));
 app.use(cors())
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
-app.use(cookieParser());
 app.use(express.static(path.join(__dirname,'../', 'frontend', 'build')));
 
+app.use('/auth', authRouter)
 app.use('/recipegen', recipegenRouter);
 app.use('/ingredient', ingredientRouter);
 
 app.get('*', async (req, res) => {
     res.sendFile(path.join(__dirname,'..', 'frontend', 'build', 'index.html'))
 })
 
-// catch 404 and forward to error handler
-app.use(function(req, res, next) {
-  next(createError(404));
-});
-
```

Comment:
Might add something like this back in later, the hack above makes this unreachable